radiomics/imageoperations.py

```diff
@@ -198,7 +198,15 @@ def checkMask(imageNode, maskNode, **kwargs):
   logger.debug('Calculating bounding box')
   # Determine bounds
   lsif = sitk.LabelStatisticsImageFilter()
-  lsif.Execute(imageNode, maskNode)
+  try:
+    lsif.Execute(imageNode, maskNode)
+  except RuntimeError as e:
+    if "Both images for LabelStatisticsImageFilter don't match type or dimension!" in e.message:
+      logger.error('Image/Mask datatype or size mismatch. Potential solution: enable resampling')
```

Comment:
I think it would be very helpful to point to the specific section of documentation on how this can be done!